cot-macros/tests/ui/derive_from_request_parts.rs

```diff
@@ -0,0 +1,143 @@
+use cot::axum::extract::RequestParts;
```

Comment:
The tests in this file are not run; note that the test coverage for the patch is 0%. The files in the `cot-macros/tests/ui` directory are used in UI/compile tests registered here: https://github.com/cot-rs/cot/blob/f9d7c43c639b9074efa9870bab1edfd96c50b29a/cot-macros/tests/compile_tests.rs

You probably want to move this file up one directory (to `cot-macros/tests`) so that they are run, or/and add some compile tests, similarly to other files in the directory.